pkg/validate/validate.go

```diff
@@ -5,39 +5,21 @@ import (
 	"fmt"
 	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha2"
 	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-// allValidatorsInSequence is used to register all SpinnakerValidator in the right execution order
-var allValidatorsInSequence []SpinnakerValidator
-
-// accountValidators keeps track of all existing AccountValidator
-var accountValidators []AccountValidator
-
-func init() {
-	accountValidators = []AccountValidator{&kubernetesAccountValidator{}}
-	allValidatorsInSequence = []SpinnakerValidator{
-		&singleNamespaceValidator{},
```

Comment:
I think we need to keep a clean way of seeing the full sequence for which validators will run. With thew new approach, it's not possible to tell easily what will be run when without looking at the implementation. Also for adding new validators, it would be easier to insert in the desired sequence.

Reply:
It's basically singleNamespace then everything else. I'd rather keep the simplicity and if we do need more logic, add a more formal validation workflow. As you can see, we cannot define things statically now as validators are determined at runtime.